site/content/docs/0.1/getting-started/introduction.md

```diff
@@ -103,7 +103,7 @@ You can also use the CDN to fetch any of our [additional builds listed in the Co
 - Read a bit more about some [important global environment settings](#important-globals) that Boosted utilizes.
```

Comment:
Do we remove all the Boosted occurrences or do we keep it for after that? There could be more than one.

Reply:
Yes, it was because I had to change the reference of the link in the homepage, this `site/content/docs/0.1/getting-started/introduction.md` is not handled yet.